### PR TITLE
Feat/simple mint nft

### PR DESCRIPTION
**Pull request by LiYuxin44**
added a new web application on minting NFTs

### CODE REVIEW

File: simple-nft-svelte/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Svelte + Vite
```
Comment (timothymcmackin) [inline simple-nft-svelte/README.md:1]: Write a simple explanation of what this app does with a link to the tutorial as in https://github.com/trilitech/tutorial-applications/blob/main/nft-taquito/README.md

File: simple-nft-svelte/contract/simple-nft.arl [hunk 2]
```diff
@@ -0,0 +1,157 @@
+// Simplified NFT contract for dApp tutorial
```
Comment (timothymcmackin) [inline simple-nft-svelte/contract/simple-nft.arl:1]: I think we're not using this file, right?

File: simple-nft-svelte/src/lib/Counter.svelte [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<script>
```
Comment (timothymcmackin) [inline simple-nft-svelte/src/lib/Counter.svelte:1]: This file is not needed

**Review by timothymcmackin — CHANGES_REQUESTED** [review]
Looks good, just write a readme and remove some unneded files like public/vite.svg and assets/svelte.svg and the old archetype contract.

**Comment by LiYuxin44** [comment]
All comments fixed.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
-# Tezos tutorial applications
+# Tutorial application: Mint NFTs from a web application
```
Comment (timothymcmackin) [inline README.md:1]: This replaces the readme at the root of tutorial-applications with a readme that is specific to the new application. Please restore this file and put the readme for the new application in /simple-nft-svelte/README.md.

File: simple-nft-svelte/src/App.svelte [hunk 5]
```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  import { BeaconWallet } from "@taquito/beacon-wallet";
+  import { NetworkType } from "@airgap/beacon-sdk";
+  import { TezosToolkit, MichelsonMap} from "@taquito/taquito";
+ 
+
+  const rpcUrl = "https://ghostnet.ecadinfra.com";
+  const Tezos = new TezosToolkit(rpcUrl);
+  const network = NetworkType.GHOSTNET;
+  //const contractAddress = "KT1WQmXvGGNsLoXh5WaaKQamEB89YfkkcqEq";
```
Comment (timothymcmackin) [inline simple-nft-svelte/src/App.svelte:10]: Remove commented-out addresses if they are no longer being used.

**Review by timothymcmackin — CHANGES_REQUESTED** [review]
Just a few final things.

**Comment by LiYuxin44** [comment]
Thanks! The problems listed have been solved.

**Comment by LiYuxin44** [comment]
Sorry, I forgot to save that file before pushing. Now it's been changed.

File: simple-nft-svelte/README.md [hunk 6]
```diff
@@ -1,47 +1,10 @@
-# Svelte + Vite
+# Tutorial application: Mint NFTs from a web application
+This web application can interacting with the Tezos blockchain to mint Non-Fungible Tokens (NFTs). This is the completed application from the tutorial [Mint NFTs from a web application](https://docs.tezos.com/tutorials/mint-NFTs-from-a-web-application/).
```
Comment (timothymcmackin) [inline simple-nft-svelte/README.md:2]: Grammar: "This web application interacts with...

**Review by timothymcmackin — APPROVED** [review]
Great, thanks for all the fixes!

**Pull request merged**